### ISSUE TITLE
`no-attribute-formation` lint may be helpful

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#379]
How about we prohibit formations with no void attributes:

```
[] > foo
  42 > bar
```

This code should lead to a warning, since `foo` doesn't have any attributes. In Java, this would be a "utility class", which is a bad idea. Of course, we should not complain about tests.

**Comment by yegor256** [comment]
@maxonfjvipon WDYT?

**Comment by maxonfjvipon** [comment]
@yegor256 take a look at this `org.eolang.go` object:
```
[] > go
  malloc.of > identifier!
    8
    m.put m.id > [m]

  [body] > to
    try > @
      body token
      [e] >>
        if. > @
          identifier.eq e.id
          e.value
          error e
      true

    [] > token
      error > backward
        jump
          to body

      [value] > jump
        identifier > id

      [res] > forward
        error > @
          jump
            res
```

Here you can see `token` object which is passed to `body` scope.
```
go.to
  [g] # here g is token
    g.backward
```

Also check this `org.eolang.structs.map.with` object:
```
# Returns the new `map` with added object
# Replaces if there was one before.
[key value] > with
  hash-code-of key > hash!
  this. > @
    map.initialized
      with.
        origin.
          filtered.
            list entries
            (hash.eq entry.hash).not > [entry] >>
        [] >>
          ^.key > key
          ^.value > value
          ^.hash > hash
```

Here you can see auto named abstract object without void attributes which is used as kind of DTO with pre-defined `key`, `value` and `hash` attributes.

Such cases are met quite often and I'm not sure we should complain about them

**Comment by yegor256** [comment]
@maxonfjvipon why can't we replace them with:

```
[body] > go-to

[] > map-initialized
```

I can't see any limitations for this. Do you?

**Comment by maxonfjvipon** [comment]
@yegor256 maybe we can but it's not in the scope of the ticket and it won't solve

**Comment by yegor256** [comment]
@maxonfjvipon I mean, we can make all objects attribute-full, can't we?

**Comment by maxonfjvipon** [comment]
@yegor256 I don't think so

**Comment by h1alexbel** [comment]
@maxonfjvipon maybe we can try this lint in experimental mode, and then try to re-write EO sources? WDYT?

**Comment by yegor256** [comment]
@h1alexbel good idea about "experimental" mode. Let's try to create such a flag for all our defects: `Defect.experimental()`. Then, eo-maven-plugin will not fail the build if such defects are found. WDYT? Let's make a ticket?

**Comment by h1alexbel** [comment]
@yegor256 extracted to #476
